images/arrow.svg

```diff
@@ -1,3 +1,22 @@
+<!--
+  Copyright 2021 Concurrent Technologies Corporation, Nteligen LLC
+
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+

```

Comment:
I'm not sure if you can claim copyright on CC0 works. I suspect this probably just needs some kind of CC0 header? I'm not familiar with CC0 and exactly what needs to be done to appease ASF.

Alternatively, we could try to find an arrow that is ALv2 license

Reply:
Looking into how to mark it  now...

Reply:
An example from Spark

https://github.com/apache/spark/blob/0494dc90af48ce7da0625485a4dc6917a244d580/data/mllib/images/origin/license.txt

Reply:
An SVG would be a better example though..

Reply:
OK.  Take a look at arrow header and the Licenses file now...

Reply:
Looks good :+1: